book/src/pong-tutorial/pong-tutorial-02.md

```diff
@@ -678,7 +678,7 @@ If all is well, we should get something that looks like this:
 In the next chapter, we'll explore the "S" in ECS and actually get these paddles
 moving!
 
-[sb]: https://slide-rs.github.io/specs/
+[sb]: https://specs.amethyst.rs/docs/tutorials/
 [sb-storage]: https://slide-rs.github.io/specs/05_storages.html#densevecstorage
 [2d]: https://docs-src.amethyst.rs/stable/amethyst_renderer/struct.Camera.html#method.standard_2d
```

Comment:
This one is 404ing too. s/amethyst_renderer/amethyst_rendy/ in the book maybe?

Reply:
Ah, if possible could you add this change to your PR? @krehera

Reply:
It'd be awesome if you fixed the few other occurences:
```[ldesgoui@desktop:~/amethyst]$ rg slide-rs
docs/FEATURES.md
31:[Specs]: https://github.com/slide-rs/specs

book/src/intro.md
115:[specs]: https://github.com/slide-rs/specs

book/src/concepts/entity_and_component.md
153:and the ["Storages" section](https://slide-rs.github.io/specs/05_storages.html) of the specs book.

book/src/pong-tutorial/pong-tutorial-02.md
681:[sb]: https://slide-rs.github.io/specs/
682:[sb-storage]: https://slide-rs.github.io/specs/05_storages.html#densevecstorage
```